src/store/wallet/wallet-selectors.ts

```diff
@@ -1,29 +1,45 @@
 import { useEffect, useMemo, useState } from 'react';
 import { useSelector } from 'react-redux';
 
-import { emptyWalletAccount } from '../../interfaces/wallet-account.interface';
+import { emptyWalletAccountState } from '../../interfaces/wallet-account-state.interface';
+import { WalletAccountInterface } from '../../interfaces/wallet-account.interface';
 import { TEZ_TOKEN_METADATA } from '../../token/data/tokens-metadata';
 import { emptyTokenMetadata } from '../../token/interfaces/token-metadata.interface';
 import { emptyToken, TokenInterface } from '../../token/interfaces/token.interface';
 import { WalletRootState, WalletState } from './wallet-state';
 
 export const useHdAccountsListSelector = () =>
-  useSelector<WalletRootState, WalletState['hdAccounts']>(({ wallet }) => wallet.hdAccounts);
+  useSelector<WalletRootState, WalletAccountInterface[]>(({ wallet }) =>
+    wallet.hdAccounts.map(
+      ({ activityGroups: _activityGroups, pendingActivities: _pendingActivities, ...restProps }) => restProps
```

Comment:
why? `_activityGroups` && `_pendingActivities`

Reply:
1. We don't need them in result
2. We don't need warnings about unused variables